content/en/docs/concepts/configuration/secret.md

```diff
@@ -633,26 +708,25 @@ This example illustrates a pod which consumes a secret containing prod
 credentials and another pod which consumes a secret with test environment
 credentials.
 
-Make the secrets:
-
+Make the kustomization.yaml with SecretGenerator
 ```shell
-$ kubectl create secret generic prod-db-secret --from-literal=username=produser --from-literal=password=Y4nys7f11
-secret "prod-db-secret" created
-$ kubectl create secret generic test-db-secret --from-literal=username=testuser --from-literal=password=iluvtests
-secret "test-db-secret" created
+$ cat <<EOF > kustomization.yaml
+secretGenerator:
+- name: prod-db-secret
+  literals:
+  - username=produser
+  - password=Y4nys7f11
+- name: test-db-secret
+  literals:
+  - username=testuser
+  - password=iluvtests
+EOF
 ```
-{{< note >}}
-Special characters such as `$`, `\*`, and `!` require escaping.
-If the password you are using has special characters, you need to escape them using the `\\` character. For example, if your actual password is `S!B\*d$zDsb`, you should execute the command this way:
-
-    kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password=S\\!B\\\*d\\$zDsb
-
-You do not need to escape special characters in passwords from files (`--from-file`).
-{{< /note >}}
```

Comment:
any particular reason why this note was removed ? if i understand correctly, those who won't use the new sub-command will still need to use the above, am i missing something ?

Reply:
or is that in 1.14 we are saying `kustomization` is the only way to go from `kubectl` standpoint ?

Reply:
`kustomization.yaml` is recommended in one of the many ways. I'll add this note back.

Reply:
updated